bpf/init.sh

```diff
@@ -496,7 +496,7 @@ for iface in $(ip -o -a l | awk '{print $2}' | cut -d: -f1 | cut -d@ -f1 | grep
 	done
 	$found && continue
 	for where in ingress egress; do
-		if tc filter show dev "$iface" "$where" | grep -q "bpf_netdev[^\.]*.o"; then
+		if tc filter show dev "$iface" "$where" | grep -Fq "bpf_netdev_${iface}.o"; then
```

Comment:
Is there a reason for not using `bpf_netdev*.o`?

Seems somewhat more future-proof in case we change the scheme again.

CCing @brb who introduced this pattern in https://github.com/cilium/cilium/commit/ed5926423cd20c1030d655a46ba7c05ce2a78eb4

Reply:
just filtering for bpf_netdev should be fine, I guess

see also:
https://github.com/cilium/cilium/blob/a8e3fa283cf40b330035265bf83d63800b296bcf/cilium/cmd/cleanup.go#L487-L495

Reply:
Thanks!

Reply:
:+1: